Build Failure on macOS 15: bindgen Cannot Find `stdlib.h`

## Description

On **macOS 15.4 (Sequoia, Apple Silicon)**, building the `zkm-recursion-gnark-ffi` crate fails during the bindgen stage because clang cannot find standard C headers (`stdlib.h`).

The Go `c-archive` builds successfully, but bindgen fails when parsing the generated header files.

### Error Output

`➜  Ziren git:(main) ✗ cargo b    
   Compiling unicode-ident v1.0.20
   Compiling quote v1.0.41
   Compiling proc-macro2 v1.0.103
   Compiling libc v0.2.177
   Compiling cfg-if v1.0.4
   Compiling serde_core v1.0.228
   Compiling serde v1.0.228
   Compiling once_cell v1.21.3
   Compiling autocfg v1.5.0
   Compiling zerocopy v0.8.27
   Compiling version_check v0.9.5
   Compiling pin-project-lite v0.2.16
   Compiling itoa v1.0.15
   Compiling tracing-core v0.1.34
   Compiling const-oid v0.9.6
   Compiling generic-array v0.14.9
   Compiling num-traits v0.2.19
   Compiling syn v1.0.109
   Compiling log v0.4.28
   Compiling scopeguard v1.2.0
   Compiling lock_api v0.4.14
   Compiling equivalent v1.0.2
   Compiling subtle v2.6.1
   Compiling rustversion v1.0.22
   Compiling typenum v1.19.0
   Compiling crossbeam-utils v0.8.21
   Compiling rayon-core v1.13.0
   Compiling hashbrown v0.16.0
   Compiling memchr v2.7.6
   Compiling indexmap v2.12.0
   Compiling crunchy v0.2.4
   Compiling syn v2.0.108
   Compiling ryu v1.0.20
   Compiling base16ct v0.2.0
   Compiling byteorder v1.5.0
   Compiling getrandom v0.2.16
   Compiling cpufeatures v0.2.17
   Compiling tiny-keccak v2.0.2
   Compiling rand_core v0.6.4
   Compiling radium v0.7.0
   Compiling tap v1.0.1
   Compiling either v1.15.0
   Compiling wyz v0.5.1
   Compiling funty v2.0.0
   Compiling ppv-lite86 v0.2.21
   Compiling rand_chacha v0.3.1
   Compiling num-integer v0.1.46
   Compiling rand v0.8.5
   Compiling unicode-xid v0.2.6
   Compiling winnow v0.7.13
   Compiling num-bigint v0.3.3
   Compiling heck v0.5.0
   Compiling strength_reduce v0.2.4
   Compiling regex-syntax v0.8.8
   Compiling transpose v0.2.3
   Compiling thiserror v1.0.69
   Compiling shlex v1.3.0
   Compiling crossbeam-epoch v0.9.18
   Compiling jobserver v0.1.34
   Compiling find-msvc-tools v0.1.4
   Compiling crossbeam-deque v0.8.6
   Compiling bitflags v2.10.0
   Compiling cc v1.2.43
   Compiling base64ct v1.8.0
   Compiling pem-rfc7468 v0.7.0
   Compiling static_assertions v1.1.0
   Compiling core-foundation-sys v0.8.7
   Compiling serde_derive v1.0.228
   Compiling tracing-attributes v0.1.30
   Compiling zeroize_derive v1.4.2
   Compiling thiserror-impl v1.0.69
   Compiling num-bigint v0.4.6
   Compiling zeroize v1.8.2
   Compiling serde_json v1.0.145
   Compiling toml_parser v1.0.4
   Compiling tracing v0.1.41
   Compiling toml_datetime v0.7.3
   Compiling toml_edit v0.23.7
   Compiling bitvec v1.0.1
   Compiling paste v1.0.15
   Compiling rayon v1.11.0
   Compiling proc-macro-crate v3.4.0
   Compiling der v0.7.10
   Compiling addchain v0.2.0
   Compiling ff_derive v0.13.1
   Compiling crypto-common v0.1.6
   Compiling block-buffer v0.10.4
   Compiling digest v0.10.7
   Compiling crypto-bigint v0.5.5
   Compiling hmac v0.12.1
   Compiling sha2 v0.10.9
   Compiling spki v0.7.3
   Compiling pkcs8 v0.10.2
   Compiling hkdf v0.12.4
   Compiling sec1 v0.7.3
   Compiling ff v0.13.1
   Compiling rfc6979 v0.4.0
   Compiling signature v2.2.0
   Compiling parity-scale-codec v3.7.5
   Compiling const_format_proc_macros v0.2.34
   Compiling itertools v0.13.0
   Compiling errno v0.3.14
   Compiling futures-core v0.3.31
   Compiling parity-scale-codec-derive v3.7.5
   Compiling const_format v0.2.35
   Compiling impl-trait-for-tuples v0.2.3
   Compiling byte-slice-cast v1.2.3
   Compiling rustix v1.1.2
   Compiling p3-maybe-rayon v0.1.0 (https://github.com/ProjectZKM/Plonky3#faa24ca4)
   Compiling nums v0.1.0
   Compiling aho-corasick v1.1.4
   Compiling strsim v0.11.1
   Compiling fastrand v2.3.0
   Compiling derive_more-impl v1.0.0
   Compiling futures-sink v0.3.31
   Compiling scale-info-derive v2.11.6
   Compiling regex-automata v0.4.13
   Compiling bytes v1.10.1
   Compiling smallvec v1.15.1
   Compiling arrayvec v0.7.6
   Compiling p3-util v0.1.0 (https://github.com/ProjectZKM/Plonky3#faa24ca4)
   Compiling hex v0.4.3
   Compiling derive_more v1.0.0
   Compiling p3-field v0.1.0 (https://github.com/ProjectZKM/Plonky3#faa24ca4)
   Compiling parking_lot_core v0.9.12
   Compiling group v0.13.0
   Compiling gcd v2.3.0
   Compiling elliptic-curve v0.13.8
   Compiling futures-channel v0.3.31
   Compiling ecdsa v0.16.9
   Compiling fnv v1.0.7
   Compiling p3-matrix v0.1.0 (https://github.com/ProjectZKM/Plonky3#faa24ca4)
   Compiling strum_macros v0.26.4
   Compiling k256 v0.13.4
   Compiling p3-symmetric v0.1.0 (https://github.com/ProjectZKM/Plonky3#faa24ca4)
   Compiling getrandom v0.3.4
   Compiling spin v0.9.8
   Compiling lazy_static v1.5.0
   Compiling pin-utils v0.1.0
   Compiling futures-task v0.3.31
   Compiling p3-dft v0.1.0 (https://github.com/ProjectZKM/Plonky3#faa24ca4)
   Compiling toml_datetime v0.6.11
   Compiling futures-macro v0.3.31
   Compiling synstructure v0.13.2
   Compiling strum v0.26.3
   Compiling slab v0.4.11
   Compiling futures-io v0.3.31
   Compiling futures-util v0.3.31
   Compiling keccak v0.1.5
   Compiling zerofrom-derive v0.1.6
   Compiling utf8parse v0.2.2
   Compiling arrayref v0.3.9
   Compiling anstyle-parse v0.2.7
   Compiling yoke-derive v0.8.1
   Compiling is_terminal_polyfill v1.70.2
   Compiling toml_write v0.1.2
   Compiling anstyle-query v1.1.4
   Compiling colorchoice v1.0.4
   Compiling stable_deref_trait v1.2.1
   Compiling anstyle v1.0.13
   Compiling zerofrom v0.1.6
   Compiling anstream v0.6.21
   Compiling zerovec-derive v0.11.2
   Compiling semver v1.0.27
   Compiling parking_lot v0.12.5
   Compiling tracing-log v0.2.0
   Compiling thread_local v1.1.9
   Compiling foldhash v0.2.0
   Compiling nu-ansi-term v0.50.3
   Compiling p3-mds v0.1.0 (https://github.com/ProjectZKM/Plonky3#faa24ca4)
   Compiling serde_spanned v0.6.9
   Compiling displaydoc v0.2.5
   Compiling ahash v0.8.12
   Compiling clap_lex v0.7.6
   Compiling allocator-api2 v0.2.21
   Compiling clap_builder v4.5.51
   Compiling p3-poseidon2 v0.1.0 (https://github.com/ProjectZKM/Plonky3#faa24ca4)
   Compiling toml_edit v0.22.27
   Compiling sha3 v0.10.8
   Compiling tempfile v3.23.0
   Compiling regex v1.12.2
   Compiling clap_derive v4.5.49
   Compiling p3-challenger v0.1.0 (https://github.com/ProjectZKM/Plonky3#faa24ca4)
   Compiling glob v0.3.3
   Compiling ansi_term v0.12.1
   Compiling rustc-hex v2.1.0
   Compiling yoke v0.8.1
   Compiling p3-commit v0.1.0 (https://github.com/ProjectZKM/Plonky3#faa24ca4)
   Compiling p3-monty-31 v0.1.0 (https://github.com/ProjectZKM/Plonky3#faa24ca4)
   Compiling tokio-macros v2.6.0
   Compiling socket2 v0.6.1
   Compiling signal-hook-registry v1.4.6
   Compiling toml v0.8.23
   Compiling mio v1.1.0
   Compiling prettyplease v0.2.37
   Compiling clap v4.5.51
   Compiling writeable v0.6.2
   Compiling litemap v0.8.1
   Compiling tokio v1.48.0
   Compiling p3-koala-bear v0.1.0 (https://github.com/ProjectZKM/Plonky3#faa24ca4)
   Compiling bincode v1.3.3
   Compiling zkm-derive v1.2.3 (/Users/blake/CLionProjects/Ziren/crates/derive)
   Compiling heck v0.4.1
   Compiling icu_normalizer_data v2.1.1
   Compiling zkm-primitives v1.2.3 (/Users/blake/CLionProjects/Ziren/crates/primitives)
   Compiling matchers v0.2.0
   Compiling sharded-slab v0.1.7
   Compiling p3-interpolation v0.1.0 (https://github.com/ProjectZKM/Plonky3#faa24ca4)
   Compiling icu_properties_data v2.1.1
   Compiling zerovec v0.11.5
   Compiling zerotrie v0.2.3
   Compiling anyhow v1.0.100
   Compiling num-modular v0.6.1
   Compiling tracing-subscriber v0.3.20
   Compiling zkm-lib v1.2.3 (/Users/blake/CLionProjects/Ziren/crates/zkvm/lib)
   Compiling scale-info v2.11.6
   Compiling p3-fri v0.1.0 (https://github.com/ProjectZKM/Plonky3#faa24ca4)
   Compiling p3-air v0.1.0 (https://github.com/ProjectZKM/Plonky3#faa24ca4)
   Compiling rlp-derive v0.1.0
   Compiling cbindgen v0.27.0
   Compiling p3-uni-stark v0.1.0 (https://github.com/ProjectZKM/Plonky3#faa24ca4)
   Compiling p3-circle v0.1.0 (https://github.com/ProjectZKM/Plonky3#faa24ca4)
   Compiling zkm-zkvm v1.2.3 (/Users/blake/CLionProjects/Ziren/crates/zkvm/entrypoint)
   Compiling tracing-forest v0.1.6
   Compiling hashbrown v0.14.5
   Compiling p3-merkle-tree v0.1.0 (https://github.com/ProjectZKM/Plonky3#faa24ca4)
   Compiling p3-mersenne-31 v0.1.0 (https://github.com/ProjectZKM/Plonky3#faa24ca4)
   Compiling p3-keccak v0.1.0 (https://github.com/ProjectZKM/Plonky3#faa24ca4)
   Compiling rayon-scan v0.1.1
   Compiling sysinfo v0.30.13
   Compiling const-hex v1.17.0
   Compiling powerfmt v0.2.0
   Compiling eyre v0.6.12
   Compiling time-core v0.1.6
   Compiling num-conv v0.1.0
   Compiling deranged v0.5.5
   Compiling zkm-stark v1.2.3 (/Users/blake/CLionProjects/Ziren/crates/stark)
   Compiling tinystr v0.8.2
   Compiling potential_utf v0.1.4
   Compiling icu_locale_core v2.1.1
   Compiling icu_collections v2.1.1
   Compiling indenter v0.3.4
   Compiling pathdiff v0.2.3
   Compiling ff v0.12.1
   Compiling memuse v0.2.2
   Compiling object v0.37.3
   Compiling icu_provider v2.1.1
   Compiling constant_time_eq v0.3.1
   Compiling dashu-base v0.4.1
   Compiling icu_properties v2.1.1
   Compiling dashu-int v0.4.1
   Compiling blake2b_simd v1.0.3
   Compiling group v0.12.1
   Compiling num-order v1.2.0
   Compiling auto_impl v1.3.0
   Compiling libm v0.2.15
   Compiling winnow v0.5.40
   Compiling gimli v0.32.3
   Compiling adler2 v2.0.1
   Compiling utf8_iter v1.0.4
   Compiling miniz_oxide v0.8.9
   Compiling rustc_version v0.4.1
   Compiling toml_edit v0.19.15
   Compiling ruint-macro v1.2.1
   Compiling rustc-demangle v0.1.26
   Compiling addr2line v0.25.1
   Compiling camino v1.2.1
   Compiling dashu-float v0.4.3
   Compiling curve25519-dalek v4.1.3
   Compiling proc-macro-crate v1.3.1
   Compiling dashu-ratio v0.4.1
   Compiling derive_more-impl v2.0.1
   Compiling thiserror v2.0.17
   Compiling httparse v1.10.1
   Compiling tower-service v0.3.3
   Compiling dashu-macros v0.4.1
   Compiling num_enum_derive v0.5.11
   Compiling primeorder v0.13.6
   Compiling num-rational v0.4.2
   Compiling zkm-core-machine v1.2.3 (/Users/blake/CLionProjects/Ziren/crates/core/machine)
   Compiling ring v0.17.14
   Compiling bytemuck_derive v1.10.2
   Compiling thiserror-impl v2.0.17
   Compiling alloy-rlp-derive v0.3.12
   Compiling backtrace v0.3.76
   Compiling num-iter v0.1.45
   Compiling num-complex v0.4.6
   Compiling percent-encoding v2.3.2
   Compiling alloy-rlp v0.3.12
   Compiling num v0.4.3
   Compiling p256 v0.13.2
   Compiling bytemuck v1.24.0
   Compiling dashu v0.4.2
   Compiling num_enum v0.5.11
   Compiling snowbridge-amcl v1.0.2
   Compiling pasta_curves v0.4.1
   Compiling pasta_curves v0.5.1
   Compiling pairing v0.22.0
   Compiling time-macros v0.2.24
   Compiling clang-sys v1.8.1
   Compiling env_filter v0.1.4
   Compiling blake2 v0.10.6
   Compiling enum-map-derive v0.17.0
   Compiling async-trait v0.1.89
   Compiling generic-array v1.3.5
   Compiling ark-std v0.4.0
   Compiling halo2curves v0.7.0
   Compiling downcast-rs v1.2.1
   Compiling jiff v0.2.15
   Compiling ark-serialize v0.4.2
   Compiling enum-map v2.7.3
   Compiling rrs-succinct v0.1.0
   Compiling ruint v1.17.0
   Compiling time v0.3.44
   Compiling bls12_381 v0.7.1
   Compiling halo2_proofs v0.1.0
   Compiling zkm-curves v1.2.3 (/Users/blake/CLionProjects/Ziren/crates/curves)
   Compiling pairing v0.23.0
   Compiling itertools v0.10.5
   Compiling serde_arrays v0.1.0
   Compiling vec_map v0.8.2
   Compiling ark-ff-macros v0.4.2
   Compiling halo2derive v0.1.0
   Compiling derivative v2.2.0
   Compiling ark-ff-asm v0.4.2
   Compiling unroll v0.1.5
   Compiling iana-time-zone v0.1.64
   Compiling try-lock v0.2.5
   Compiling untrusted v0.9.0
   Compiling option-ext v0.2.0
   Compiling minimal-lexical v0.2.1
   Compiling nohash-hasher v0.2.0
   Compiling elf v0.7.4
   Compiling dirs-sys v0.4.1
   Compiling want v0.3.1
   Compiling chrono v0.4.42
   Compiling jubjub v0.9.0
   Compiling derive_more v2.0.1
   Compiling nom v7.1.3
   Compiling zkm-recursion-core v1.2.3 (/Users/blake/CLionProjects/Ziren/crates/recursion/core)
   Compiling p3-keccak-air v0.1.0 (https://github.com/ProjectZKM/Plonky3#faa24ca4)
   Compiling halo2 v0.1.0-beta.2
   Compiling num_cpus v1.17.0
   Compiling libloading v0.8.9
   Compiling env_logger v0.11.8
   Compiling ark-ff v0.4.2
   Compiling size v0.4.1
   Compiling zkm-core-executor v1.2.3 (/Users/blake/CLionProjects/Ziren/crates/core/executor)
   Compiling bindgen v0.70.1
   Compiling web-time v1.1.0
   Compiling httpdate v1.0.3
   Compiling alloy-primitives v1.4.1
   Compiling cexpr v0.6.0
   Compiling dirs v5.0.1
   Compiling form_urlencoded v1.2.2
   Compiling icu_normalizer v2.1.1
   Compiling rlp v0.5.2
   Compiling http v0.2.12
   Compiling rustc-hash v1.1.0
   Compiling idna_adapter v1.2.1
   Compiling base64 v0.21.7
   Compiling dunce v1.0.5
   Compiling idna v1.1.0
   Compiling impl-codec v0.6.0
   Compiling futures-executor v0.3.31
   Compiling impl-rlp v0.3.0
   Compiling sct v0.7.1
   Compiling fixed-hash v0.8.0
   Compiling open-fastrlp-derive v0.1.1
   Compiling impl-serde v0.4.0
   Compiling zkm-recursion-derive v1.2.3 (/Users/blake/CLionProjects/Ziren/crates/recursion/derive)
   Compiling ident_case v1.0.1
   Compiling darling_core v0.21.3
   Compiling url v2.5.7
   Compiling futures v0.3.31
   Compiling tokio-util v0.7.16
   Compiling http v1.3.1
   Compiling uint v0.9.5
   Compiling num_enum_derive v0.7.5
   Compiling mime v0.3.17
   Compiling sdd v3.0.10
   Compiling ipnet v2.11.0
   Compiling foldhash v0.1.5
   Compiling wasm-bindgen-shared v0.2.105
   Compiling hashbrown v0.15.5
   Compiling scc v2.4.0
   Compiling p3-bn254-fr v0.1.0 (https://github.com/ProjectZKM/Plonky3#faa24ca4)
   Compiling num_enum v0.7.5
   Compiling primitive-types v0.12.2
   Compiling ethbloom v0.13.0
   Compiling http-body v0.4.6
   Compiling cargo-platform v0.1.9
   Compiling serial_test_derive v3.2.0
   Compiling crossbeam-channel v0.5.15
   Compiling pkg-config v0.3.32
   Compiling darling_macro v0.21.3
   Compiling rustls v0.21.12
   Compiling ethereum-types v0.14.1
   Compiling serial_test v3.2.0
   Compiling tracing-appender v0.2.3
   Compiling darling v0.21.3
   Compiling cargo_metadata v0.18.1
   Compiling lru v0.12.5
   Compiling h2 v0.3.27
   Compiling rustls-webpki v0.101.7
   Compiling ring v0.16.20
   Compiling socket2 v0.5.10
   Compiling zkhash v0.2.0
   Compiling tower-layer v0.3.3
   Compiling vcpkg v0.2.15
   Compiling libz-sys v1.1.22
   Compiling zkm-recursion-gnark-ffi v1.2.3 (/Users/blake/CLionProjects/Ziren/crates/recursion/gnark-ffi)
   Compiling serde_urlencoded v0.7.1
   Compiling core-foundation v0.9.4
   Compiling itertools v0.14.0
   Compiling crc-catalog v2.4.0
   Compiling data-encoding v2.9.0
   Compiling bumpalo v3.19.0
   Compiling wasm-bindgen-macro-support v0.2.105
   Compiling crc v3.3.0
   Compiling prost-derive v0.13.5
   Compiling hyper v0.14.32
   Compiling serde_with_macros v3.15.1
   Compiling ethabi v18.0.0
   Compiling open-fastrlp v0.1.4
   Compiling http-body v1.0.1
   Compiling syn-solidity v1.4.1
   Compiling inout v0.1.4
   Compiling pin-project-internal v1.1.10
   Compiling wasm-bindgen v0.2.105
   Compiling macro-string v0.1.4
   Compiling proc-macro-error-attr2 v2.0.0
   Compiling bitflags v1.3.2
   Compiling system-configuration-sys v0.5.0
   Compiling untrusted v0.7.1
   Compiling tinyvec_macros v0.1.1
   Compiling tinyvec v1.10.0
   Compiling proc-macro-error2 v2.0.1
   Compiling pin-project v1.1.10
   Compiling zkm-recursion-compiler v1.2.3 (/Users/blake/CLionProjects/Ziren/crates/recursion/compiler)
   Compiling prost-derive v0.11.9
error: failed to run custom build command for `zkm-recursion-gnark-ffi v1.2.3 (/Users/blake/CLionProjects/Ziren/crates/recursion/gnark-ffi)`

Caused by:
  process didn't exit successfully: `/Users/blake/CLionProjects/Ziren/target/debug/build/zkm-recursion-gnark-ffi-320e3d3d5ef93584/build-script-build` (exit status: 101)
  --- stdout
  cargo:rerun-if-changed=go
  Building Go library at /Users/blake/CLionProjects/Ziren/target/debug/build/zkm-recursion-gnark-ffi-f74b27fc63ff7f2f/out/libzkmgnark.a

  --- stderr
  /Users/blake/CLionProjects/Ziren/target/debug/build/zkm-recursion-gnark-ffi-f74b27fc63ff7f2f/out/./koalabear.h:2:1: error: unknown type name 'uint32_t'
  /Users/blake/CLionProjects/Ziren/target/debug/build/zkm-recursion-gnark-ffi-f74b27fc63ff7f2f/out/./koalabear.h:2:26: error: unknown type name 'uint32_t'
  /Users/blake/CLionProjects/Ziren/target/debug/build/zkm-recursion-gnark-ffi-f74b27fc63ff7f2f/out/./koalabear.h:2:38: error: unknown type name 'uint32_t'
  /Users/blake/CLionProjects/Ziren/target/debug/build/zkm-recursion-gnark-ffi-f74b27fc63ff7f2f/out/./koalabear.h:2:50: error: unknown type name 'uint32_t'
  /Users/blake/CLionProjects/Ziren/target/debug/build/zkm-recursion-gnark-ffi-f74b27fc63ff7f2f/out/./koalabear.h:2:62: error: unknown type name 'uint32_t'
  /Users/blake/CLionProjects/Ziren/target/debug/build/zkm-recursion-gnark-ffi-f74b27fc63ff7f2f/out/./koalabear.h:2:74: error: unknown type name 'uint32_t'
  /Users/blake/CLionProjects/Ziren/target/debug/build/zkm-recursion-gnark-ffi-f74b27fc63ff7f2f/out/./koalabear.h:3:1: error: unknown type name 'uint32_t'
  /Users/blake/CLionProjects/Ziren/target/debug/build/zkm-recursion-gnark-ffi-f74b27fc63ff7f2f/out/./koalabear.h:3:23: error: unknown type name 'uint32_t'
  /Users/blake/CLionProjects/Ziren/target/debug/build/zkm-recursion-gnark-ffi-f74b27fc63ff7f2f/out/libzkmgnark.h:25:10: fatal error: 'stdlib.h' file not found

  thread 'main' (67531237) panicked at crates/recursion/gnark-ffi/build.rs:50:18:
  Unable to generate bindings: ClangDiagnostic("/Users/blake/CLionProjects/Ziren/target/debug/build/zkm-recursion-gnark-ffi-f74b27fc63ff7f2f/out/./koalabear.h:2:1: error: unknown type name 'uint32_t'\n/Users/blake/CLionProjects/Ziren/target/debug/build/zkm-recursion-gnark-ffi-f74b27fc63ff7f2f/out/./koalabear.h:2:26: error: unknown type name 'uint32_t'\n/Users/blake/CLionProjects/Ziren/target/debug/build/zkm-recursion-gnark-ffi-f74b27fc63ff7f2f/out/./koalabear.h:2:38: error: unknown type name 'uint32_t'\n/Users/blake/CLionProjects/Ziren/target/debug/build/zkm-recursion-gnark-ffi-f74b27fc63ff7f2f/out/./koalabear.h:2:50: error: unknown type name 'uint32_t'\n/Users/blake/CLionProjects/Ziren/target/debug/build/zkm-recursion-gnark-ffi-f74b27fc63ff7f2f/out/./koalabear.h:2:62: error: unknown type name 'uint32_t'\n/Users/blake/CLionProjects/Ziren/target/debug/build/zkm-recursion-gnark-ffi-f74b27fc63ff7f2f/out/./koalabear.h:2:74: error: unknown type name 'uint32_t'\n/Users/blake/CLionProjects/Ziren/target/debug/build/zkm-recursion-gnark-ffi-f74b27fc63ff7f2f/out/./koalabear.h:3:1: error: unknown type name 'uint32_t'\n/Users/blake/CLionProjects/Ziren/target/debug/build/zkm-recursion-gnark-ffi-f74b27fc63ff7f2f/out/./koalabear.h:3:23: error: unknown type name 'uint32_t'\n/Users/blake/CLionProjects/Ziren/target/debug/build/zkm-recursion-gnark-ffi-f74b27fc63ff7f2f/out/libzkmgnark.h:25:10: fatal error: 'stdlib.h' file not found\n")
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...`

### Environment

- OS: macOS 15.4 (Sequoia)
- Arch: Apple Silicon (arm64)
- Crate: crates/recursion/gnark-ffi
- Toolchain: zkm-toolchain-0827/1217

## Steps to Reproduce

From repository root:
`source ~/.zkm-toolchain/env && cargo build`

--- a/crates/recursion/gnark-ffi/build.rs
+++ b/crates/recursion/gnark-ffi/build.rs
@@ -54,7 +54,7 @@ fn main() {
 
             let bindings = bindgen::Builder::default()
                 .header(header_path.to_str().unwrap())
-                .clang_arg(format!("-I{}", include))
+                .clang_arg(format!("-I{include}"))
                 .generate()
                 .expect("Unable to generate bindings");
 

--- a/crates/recursion/gnark-ffi/build.rs
+++ b/crates/recursion/gnark-ffi/build.rs
@@ -44,8 +44,17 @@ fn main() {
 
             // Generate bindings using bindgen
             let header_path = dest_path.join(format!("lib{lib_name}.h"));
+
+            let sysroot = Command::new("rustc")
+                .args(["--print", "sysroot"])
+                .output()
+                .expect("rustc not found");
+            let sysroot = String::from_utf8(sysroot.stdout).unwrap();
+            let include = format!("{}/lib/mips-zkm-elf/include", sysroot.trim());
+
             let bindings = bindgen::Builder::default()
                 .header(header_path.to_str().unwrap())
+                .clang_arg(format!("-I{}", include))
                 .generate()
                 .expect("Unable to generate bindings");
 

